### DISCUSSION TITLE
이렇게 계속 뜨는거면 잘 되고 있는 건가요?

### DISCUSSION THREAD

**Discussion by sallyyyn** [changdoc/naver-vaccine-macro#182] [Q&A]
잘 모르겠어서ㅠㅠ 아까 뭣도 모르고 여러개 했다가 오류 떠서 1시간 기다리고 다시 하는 거거든여... 한개만 노리고 있고요! 이렇게 하면 될까요? 아까처럼 오류 뜨진 않겠죠? 
![image](https://user-images.githubusercontent.com/88305445/127808728-90420fff-7b95-4e82-9fbe-2ac928281c61.png)

**Comment by changdoc** [comment]
네 차단은 https://github.com/changdoc/naver-vaccine-macro/issues/42 이 곳 참조 하시면 되겠습니다.

**Reply by sallyyyn** [reply]
저거 돌리면서 지도 돌려도 상관없나요? 아니면 무조건 하나만 해야하나요?

**Reply by changdoc** [reply]
지도와 신청 페이지가 상관관계가 없긴한데요.
신청 페이지 접근을 짧은 시간에 여러차례 하게 되면 차단되는 형태로 지도에서 디텍팅이 되면 결국 신청 페이지로 이동하게 되어 신청 할 수 있는 타이밍에 차단을 당할 여지가 있긴합니다.

한 병원의 신청 페이지에서 돌리는 새로고침 주기를 좀 늘려서(5초정도..) 설정해두고 지도와 함께 돌리면 큰 문제가 없을거 같기도 하네요.

저도 병행 트라이는 안 해봐서 해보시고 공유해주시면 좋을 것 같네요

**Reply by sallyyyn** [reply]
지도 돌리는데 자꾸 집에서 너무 먼 곳이 걸리는데 이건 위치 조정 못하나요? 아무리 찾아봐도 없어서요ㅠㅠ 질문이 많아 죄송합니다.......

**Reply by changdoc** [reply]
네이버 지도에서 돌리고 계신거면 확대를 하시면 범위가 좁아질텐데요..

카카오 지도라면 확대 하시고 Request Header 다시 추출 하시고 동작 시키시면 될 것 같습니다.

**Reply by cjhaaei** [reply]
안녕하세요, 도움 잘 받고 있는데 궁금한점이 있습니다. 혹시 개별 병원만 공략 (1.0.9) 와 최신 버전인 지도 기반 둘중에 어떤게 성공 확률이 더 높을까요?

**Reply by sallyyyn** [reply]
카카오지도 {"bottom Right":{"x":128.74980202239436,"y":35.205712089211495},"onlyLeft":false,"order":"latitude","topLeft":{"x":128.82841527871543,"y":35.16002533499305}} 이거 넣었는데 병원이 안떠요 지금 저녁이라서 그런가요?ㅠㅠ

그리고 네이버는 아무리 확대해도 매크로에서 범위가 좁아지지 않아요. 카카오지도는 제가 딱 원하는대로 병원이 33개 나오는데 네이버는 97개가 나오네요ㅠㅠ 혹시 네이버도 서버 입력하고 (네이버가 A~Z) 카카오도 서버 입력(카카오 A~G)면 그 A~G에서만 백신 신청이 가능한가요?

**Reply by cjhaaei** [reply]
네이버 확대 하셔서 '현 지도에서 검색' 누르신 후에 새로 고침누르면 받아지는 graphql 값이용하신거 맞나요? 저는 확대해서 재검색해서 병원목록 생성하니 병원수가 달라집니다.
